src/components/2019/year-in-review/News.js

```diff
@@ -0,0 +1,534 @@
+import React from 'react'
+import s from 'styled-components'
+
+import { MoreStories, PhotoGallery, Date, SectionMainText, PrimaryStory, StyledStoryLink, SecondaryStory, PrimaryStoryLine, SubSectionHeader, MainHeader } from './Typograph'
+
+const Wrapper = s.div`
+    /* Util classes */
+    .right-border {
+        border-right: 1px solid rgba(255, 255, 255, 0.3);
+    }
+
+    .right-border-inverse {
+        border-right: 1px solid rgba(0, 0, 0, 0.3);
+    }
+
+    .narrow-col-right {
+        padding-right: 24px;
+    }
+
+    .narrow-col-left {
+        padding-left: 24px;
+    }
+`
+
+const Section = s.div`
+    background-color: ${({ inverse }) => inverse ? "white" : "#131313"};
+    padding: ${({ inverse }) => inverse ? "80px 0px" : "120px 0px"};
+
+`
+const News = () => {
+  return (
+    <Wrapper id="news">
+        <Section id="news">
```

Comment:
what's the purpose of these ids? I know atleast one is needed for the scrolling nav functionality, but what's the other one for?